parser-typechecker/src/Unison/Codebase/ShortBranchHash.hs

```diff
@@ -2,7 +2,6 @@ module Unison.Codebase.ShortBranchHash
   ( toString,
     toHash,
     fromHash,
-    fullFromHash,
```

Comment:
This combinator didn't make much sense, it was constructing a "long" ShortBranchHash. It seems as though it was created/used out of some sort of misunderstanding.